index.html

```diff
@@ -44,5 +51,141 @@ <h3>I’am a Software Developer</h3>
         <li><img src="./images/Footer icons/5.png" alt="" /></li>
       </ul>
     </section>
+
+    <section class="works">
+      <h1>My Recent Works</h1>
```

Comment:
- [ ] Kindly change this to be any heading from `h2` to `h6`, for example: `h2` since the `h1` should only exist once on your page for the main heading.